openerp/models.py

```diff
@@ -341,7 +341,7 @@ class BaseModel(object):
     # 1. includes self fields,
     # 2. uses column_info instead of a triple.
     # Warning: _all_columns is deprecated, use _fields instead
-    _all_columns = {}
+    # _all_columns = {}
```

Comment:
Why not remove that altogether, and move the comments to the property if we're keeping them?